packages/component-library/stories/LineChart.story.js

```diff
@@ -104,117 +68,242 @@ const customLegend = legendData => {
+          "Data series labels",
+          sampleDataSeriesLabel,
+          GROUP_IDS.DATA
+        );
+        const data = object("Data", sampleTransportationData, GROUP_IDS.DATA);
+
+        return (
+          <LineChart
+            data={data}
+            dataKey={dataKey}
+            dataValue={dataValue}
+            dataSeries={dataSeries}
+            dataSeriesLabel={dataSeriesLabel}
+            subtitle={subtitle}
+            title={title}
+            xLabel={xLabel}
+            yLabel={yLabel}
+            xNumberFormatter={x => civicFormat[optionSelectX](x)}
+            yNumberFormatter={y => civicFormat[optionSelectY](y)}
+          />
+        );
+      },
+      { notes }
+    )
+    .add(
+      "Custom",
+      () => {
+        const sampleTransportationData = [
+          { x: 2001, y: 217309, series: "weekday" },
+          { x: 2002, y: 254651, series: "weekday" },
+          { x: 2003, y: 213986, series: "weekday" },
+          { x: 2004, y: 266412, series: "weekday" },
+          { x: 2005, y: 313494, series: "weekday" },
+          { x: 2001, y: 73028, series: "saturday" },
+          { x: 2002, y: 113778, series: "saturday" },
+          { x: 2003, y: 111636, series: "saturday" },
+          { x: 2004, y: 155272, series: "saturday" },
+          { x: 2005, y: 188757, series: "saturday" },
+          { x: 2001, y: 73028, series: "sunday" },
+          { x: 2002, y: 75042, series: "sunday" },
+          { x: 2003, y: 75615, series: "sunday" },
+          { x: 2004, y: 108040, series: "sunday" },
+          { x: 2005, y: 134569, series: "sunday" }
+        ];
+        const sampleDataSeries = "series";
+        const sampleDataSeriesLabel = [
+          { category: "weekday", label: "Weekday" },
+          { category: "saturday", label: "Saturday" },
+          { category: "sunday", label: "Sunday" }
+        ];
+        const sampleDomain = { x: [2001, 2005], y: [0, 350000] };
+        // const sampleSize = { key: "y" };
+
+        const title = text(
+          "Title",
+          "Public Transit Ridership",
+          GROUP_IDS.LABELS
+        );
+        const subtitle = text(
+          "Subtitle",
+          "Average daily ridership for TriMet bus and rail (unlinked trips)",
+          GROUP_IDS.LABELS
+        );
+        const xLabel = text("X-axis label", "Year", GROUP_IDS.LABELS);
+        const optionSelectX = options(
+          "X-axis value format",
+          xFormatterOptions,
+          "year",
+          { display: "select" },
+          GROUP_IDS.LABELS
+        );
+        const yLabel = text("Y-axis label", "Ridership", GROUP_IDS.LABELS);
+        const optionSelectY = options(
+          "Y-axis value format",
+          yFormatterOptions,
+          "numeric",
+          { display: "select" },
+          GROUP_IDS.LABELS
+        );
+        const dataKey = text("Data key", "x", GROUP_IDS.DATA);
+        const dataKeyLabel = text("Data key label", "Year", GROUP_IDS.DATA);
```

Comment:
I am receiving a `propTypes` error stating the the `dataKeyLabel` is expecting an array.